modules/openapi-generator/src/main/resources/csharp-netcore/appveyor.mustache

```diff
@@ -0,0 +1,13 @@
+# auto-generated by OpenAPI Generator (https://github.com/OpenAPITools/openapi-generator)
+#
+image: Visual Studio 2019
+clone_depth: 1
+before_build:
+- dotnet restore
```

Comment:
```suggestion
```

If you use the `dotnet build` command, `dotnet restore` is automatic and is not needed.